benchmarks/naslib/problem.py

```diff
@@ -0,0 +1,98 @@
+import os
+import sys
+from typing import Any
+from typing import List
+
+from kurobako import problem
+
+
+sys.stdout = open(os.devnull, "w")  # Suppress output
```

Comment:
Since the current implementation requires only `get_dataset_api` of naslib, we may remove the hack on suppressing stdout.
```suggestion
```

[`get_dataset_api`](https://github.com/automl/NASLib/blob/Develop/naslib/utils/get_dataset_api.py) does not show any message on stdout as follows.
```python
>>> import naslib
>>> from naslib.utils import get_dataset_api 
>>> 
```